src/index.html

```diff
@@ -1,13 +1,280 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakerlab</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
     <script type="text/javascript" src="scripts/main.js"></script>
+    <link rel="icon" type="image/x-icon"
+      href="./images/icon/favicon.png"
+    >
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img src="images/icon/logo.png" alt="Bakerlab"
+                class="top-bar__logo">
+            </a>
+
+            <div class="top-bar__desctop">
+              <nav>
+                <ul class="nav__list">
+                  <li class="nav__link">
+                    <a class="nav__link" href="#what-we-bake">Products</a>
+                  </li>
+
+                  <li class="nav__link">
+                    <a class="nav__link" href="#about-us">About us</a>
+                  </li>
+
+                  <li class="nav__link">
+                    <a class="nav__link" href="#">Blog</a>
+                  </li>
+
+                  <li class="nav__link nav__link--contact-us">
+                    <div class="nav__contacts-us">
+                      <img src="./images/icon/Icon-phone.svg" alt="icon-phone">
+                      <a class="nav__link--contact-us" href="#contact-us">CONTACT US</a>
+                    </div>
+                  </li>
+                </ul>
+              </nav>
+            </div>
+            <div class="top-bar__icons">
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h3 class="header__slogan">We bake your fantasies — croissants, cakes, cookies, and cupcakes.</h3>
+            <h1 class="header__title">Creative bakery</h1>
+            <img
+              class="header__image"
+              src="./images/cookie.png"
+              alt="cokkie"
+            >
+            <button class="button header__button">
+              <img class="button__icon" src="./images/icon/cookie-button-icon.png"
+                alt="icon-cookie">
```

Comment:
Need to check and fix it everywhere
```suggestion
              <img
                class="button__icon" 
                src="./images/icon/cookie-button-icon.png"
                alt="icon-cookie"
              >
```